### PR TITLE
fix: set args for Retry action in chat editing widget

### PR DESCRIPTION
**Pull request by joyceerhl**

### CODE REVIEW

File: src/vs/workbench/contrib/chat/browser/actions/chatTitleActions.ts [hunk 1]
```diff
@@ -208,13 +208,17 @@ export function registerChatTitleActions() {
 		}
 
 		async run(accessor: ServicesAccessor, ...args: any[]) {
-			const item = args[0];
+			const chatWidgetService = accessor.get(IChatWidgetService);
+
+			let item = args[0];
+			if (typeof item === 'object' && !!item && 'sessionId' in item) {
```
Comment (roblourens) [inline src/vs/workbench/contrib/chat/browser/actions/chatTitleActions.ts:214]: Why not `isResponseVM`? Can we move away from these kinds of checks?

File: src/vs/workbench/contrib/chat/browser/chatInputPart.ts [hunk 2]
```diff
@@ -993,6 +993,9 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 
 		this._chatEditsActionsDisposables.add(this.instantiationService.createInstance(MenuWorkbenchButtonBar, actionsContainer, MenuId.ChatEditingWidgetToolbar, {
 			telemetrySource: this.options.menus.telemetrySource,
+			menuOptions: {
```
Comment (roblourens) [inline src/vs/workbench/contrib/chat/browser/chatInputPart.ts:996]: Is this guaranteed to rerender when the session id changes?

Comment (joyceerhl) [reply src/vs/workbench/contrib/chat/browser/chatInputPart.ts:996]: Yeah, if anything it's rerendering a bit too much right now, gotta fix that in main too

File: src/vs/workbench/contrib/chat/browser/actions/chatTitleActions.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (joyceerhl) [reply src/vs/workbench/contrib/chat/browser/actions/chatTitleActions.ts:214]: https://github.com/microsoft/vscode/pull/232645

**Review by Yoyokrazy — APPROVED** [review]

**Review by roblourens — APPROVED** [review]

**Pull request merged**